src/gui/main.cpp

```diff
@@ -452,12 +452,12 @@ int main(int argc, char **argv)
     setupLogging(options);
 
 #if QT_VERSION >= QT_VERSION_CHECK(6, 3, 0)
-    qCDebug(lcMain) << QNetworkInformation::availableBackends().join(QStringLiteral(", "));
     if (!QNetworkInformation::loadDefaultBackend()) {
-        qCWarning(lcMain) << "Failed to load QNetworkInformation";
+        qCWarning(lcMain) << "Failed to load QNetworkInformation. Available backends:" << QNetworkInformation::availableBackends().join(QStringLiteral(", "));
```

Comment:
We should also implement the fallback from https://github.com/qt/qtbase/commit/ce4531a4901504f62f8c76c3542d3b1aad2b384a#diff-db55ac8df386b2d509573b97fbdc986e20426e4b9bcf0fbc7e03a866406b1631R685 ?
In my tests this will load the glib plugin.

Reply:
The default network manager relies on the availability of a dbus service. Which obviously is not available in my wsl setup.